src/figuresClasses.ts

```diff
@@ -1,11 +1,74 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+    public shape: string = 'triangle',
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Sides must be greater than 0');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error(
+        'The longest side of a triangle must be less than' +
+          'the sum of the other two sides',
+      );
```

Comment:
[CODE STYLE] - The error message should be more descriptive to clearly indicate the problem. Instead of a general 'Sides must be greater than 0', you could specify 'All sides of a triangle must be greater than 0'. Similarly, for the sum of sides condition, a more descriptive message would be beneficial.